### ISSUE TITLE
Spells macros and flow break with this mod

### ISSUE THREAD

**Issue by NdranC** [caewok/fvtt-walled-templates#115]
I'm getting all kinds of errors when I try to use this module in a midi-qol environment. When I try to place a template in a scene as a result of a spell the console gets spammed by errors and the logic of applying damage and saving throws doesn't continue.

![image](https://github.com/caewok/fvtt-walled-templates/assets/27837113/3c7b8fb8-9021-40b8-89c6-3e510e23e3a8)
![image](https://github.com/caewok/fvtt-walled-templates/assets/27837113/600f59ab-843a-49b2-9b48-8b12ca421a47)

This might be a problem with midi-qol as well but it's easier to disable this module and test that everything works fine. I can say that when I place templates without any attached logic from the template tool no errors seem to occur.

**Comment by caewok** [comment]
It is reporting that your token has no actor. That seems incorrect and not likely due to Walled Templates. Does this happen if you create a new scene and add a new actor to it?

**Comment by NdranC** [comment]
So, this issue was only happening when enabling Walled Templates. I installed the mod because of Gambit's attacks of opportunity feature. The funny thing is that after updating all the mods and trying to replicate the issue today... I can't. I'm not sure what changed as I'm almost certain my mod configuration is the same and went I tested this a couple days ago I already had the most recent version of all mods, regardless I can't replicate this any longer.

If it happens again I'll reopen the issue. Thank you.
